lib/common/tests/flags/Makefile.am

```diff
@@ -11,7 +11,7 @@ AM_CPPFLAGS = -I$(top_srcdir)/include		\
 	      -I$(top_builddir)/include		\
 	      -I$(top_srcdir)/lib/common
 LDADD = $(top_builddir)/lib/common/libcrmcommon_test.la
-AM_LDFLAGS = $(LDFLAGS_WRAP)
+AM_LDFLAGS =
```

Comment:
Crazy!

We should update Pacemaker Development to say when to set this and when not to

Reply:
Well I don't completely understand what is going on here, but it looks like we're safe if we only set `AM_LDFLAGS = $(LDFLAGS_WRAP)` for those tests that actually use a wrapped function.  I've expanded the patch to make that the case, so at least it is consistent.  I'll add a blurb to the docs as well.